### ISSUE TITLE
Infinite resursion on cycle references

### ISSUE THREAD

**Issue by cen1** [krishna81m/jackson-nested-prop-filter#5]
If class A references class B and class B references class A this results in stackoverflow. Method `getBeanUtilsNestedJsonAttribute` should somehow keep track of the nesting level.

**Comment by manojkr001** [comment]
Hi @cen1, Did you get any work around or a fix for this stackoverflow exception for cyclic references

**Comment by cen1** [comment]
@manojkr001 no, I no longer use this lbrary. I just let clients filter on their own using https://github.com/bohnman/squiggly

**Comment by manojkr001** [comment]
@cen1 Thanks for the update and information.

**Comment by krishna81m** [comment]
Glad swiggly open sourced the full implementation.
